### PR TITLE
[de] Translation Chapter 1

### PR DESCRIPTION
**Pull request by MarcusFra**
The german chapter 1 is ready to be merged. Again, I am really sorry for inconveniences caused by my delay. 
Regarding chapter 1 I'd like to exchange some thoughts before being merged. @JesperDramsch and @fabridamicelli, I'd be happy to hear about your opinions:

- "Natural Language Processing": Here I am a bit unsure if it would not be better to let Natural Language Processing (NLP) instead of translating it. Though "Computerlinguistik" is often used, I've also encountered different translations as "Maschinelle Sprachverarbeitung", and I think NLP would be a more intuitive abbreviation than CL. 
- I would favor "Feintuning" or "feintunen" instead of "Fein-Tuning" or "fein-tunen".
- I left "Question Answering" as is. I also know "Frage-Antwort-System", but especially in the context of distinguishing generative or extractive QA my first thought is better to leave it.  

Depending on what you prefer, I will adjust it accordingly.
 
Of course, I also would be thankful if you like to revise chapter 1. I already read chapter 3 and the current parts of chapter 4, and after merging chapter 1 I would open a PR with minor changes /suggestions to your chapters. Would that be in your interest?

### CODE REVIEW

**Comment by JesperDramsch** [comment]
> "Natural Language Processing": Here I am a bit unsure if it would not be better to let Natural Language Processing (NLP) instead of translating it. Though "Computerlinguistik" is often used, I've also encountered different translations as "Maschinelle Sprachverarbeitung", and I think NLP would be a more intuitive abbreviation than CL.

I don't have a strong opinion here. I understand why someone would use NLP. But I think we have to also take into account existing terminology in Germany. If someone takes this course and "Computerlinguistik" is not mentioned at any place, when it's the name of institutes / research groups, I believe that would be a miss on our part. I'd be happy to use CL and MSV interchangeably. It might also be worth adding a small paragraph in the first chapter clarifying our use of either CL or MSV, but that NLP is used more and more as an anglicism in the German language. I think it makes most sense for @lewtun to make that call.

> I would favor "Feintuning" or "feintunen" instead of "Fein-Tuning" or "fein-tunen".

Either is fine by me, I just don't have the time to make these changes. I thought there was a grammatical rule about hyphenation with words that have an English root, but the Duden has "Feintuning", so that works as long as we are consistent.

> I left "Question Answering" as is. I also know "Frage-Antwort-System", but especially in the context of distinguishing generative or extractive QA my first thought is better to leave it.

I would translate it. In the end, I thought this course should be accessible to someone without English knowledge and you lose context by leaving it as-is, when there is a perfectly fine translation.

Again I don't have strong opinions on this, as this is something I worked on half a year ago ( #43 ) and I don't really have time right now to spend cycles on this currently, but I figured I'd answer the direct suggestions / questions with some thoughts.

**Comment by MarcusFra** [comment]
Thank you @JesperDramsch! Yes, I think it's a bit a question of preference as some prefer the english terminology for tasks and architectural aspects since in practice it's often used. But I totally get your point and for me either is fine, too. Regardless for what we go for, I'd also name the other words used in German in the first appearance. 

>> I would favor "Feintuning" or "feintunen" instead of "Fein-Tuning" or "fein-tunen".

> Either is fine by me, I just don't have the time to make these changes. I thought there was a grammatical rule about hyphenation with words that have an English root, but the Duden has "Feintuning", so that works as long as we are consistent.

Yes, I think Feintuning is the correct version in German.

I'll wait for the other opinions, and if changes to your files need to be made I will take care of it and ensure that it is consistent.

**Comment by JesperDramsch** [comment]
Dope thanks! It's probably just a replace-all but nice and clean in a single commit.

**Comment by fabridamicelli** [comment]
> The german chapter 1 is ready to be merged. Again, I am really sorry for inconveniences caused by my delay. Regarding chapter 1 I'd like to exchange some thoughts before being merged. @JesperDramsch and @fabridamicelli, I'd be happy to hear about your opinions:
> 
> * "Natural Language Processing": Here I am a bit unsure if it would not be better to let Natural Language Processing (NLP) instead of translating it. Though "Computerlinguistik" is often used, I've also encountered different translations as "Maschinelle Sprachverarbeitung", and I think NLP would be a more intuitive abbreviation than CL.
> * I would favor "Feintuning" or "feintunen" instead of "Fein-Tuning" or "fein-tunen".
> * I left "Question Answering" as is. I also know "Frage-Antwort-System", but especially in the context of distinguishing generative or extractive QA my first thought is better to leave it.
> 
> Depending on what you prefer, I will adjust it accordingly.
> 
> Of course, I also would be thankful if you like to revise chapter 1. I already read chapter 3 and the current parts of chapter 4, and after merging chapter 1 I would open a PR with minor changes /suggestions to your chapters. Would that be in your interest?

I am also fine with your choices and agree with Jesper that we should make the material more accessible to non-english speakers so IMO having a couple of sentences on that cannot hurt (as Jesper suggested). After that, as long as we stay consistent it should be fine.

I am happy to take any suggestions you might have on the parts I translated, so feel free to open a PR on that whenever you have time to take a look at that.

**Comment by lewtun** [comment]
Hi @MarcusFra thanks so much for this epic translation 💪 !!

Regarding your questions:

> "Natural Language Processing": Here I am a bit unsure if it would not be better to let Natural Language Processing (NLP) instead of translating it. Though "Computerlinguistik" is often used, I've also encountered different translations as "Maschinelle Sprachverarbeitung", and I think NLP would be a more intuitive abbreviation than CL.

One thing we've done in other translations is to provide a pointer to the English meaning where applicable, e.g. "Computerlinguistik (Natural Language Processing oder NLP auf Englisch)"

I don't have a strong opinion on the matter, so am happy to follow the advice from @JesperDramsch to make the material as accessible as possible to native German speakers. A similar comment applies to the other questions, where the consensus seems to be on using German words / phrases as much as possible :)

Feel free to ping me when this is ready for a final check / merge!

File: chapters/de/chapter1/1.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Einführung
+
```
Comment (lewtun) [inline chapters/de/chapter1/1.mdx:2]: We recently added some meta content to the English version, so you'll need to add things like the following in each section:

```suggestion

<CourseFloatingBanner
    chapter={1}
    classNames="absolute z-10 right-0 top-0"
/>

```

Check out the corresponding English files for the snippet you can copy-paste :)

File: chapters/de/chapter1/3.mdx [hunk 2]
```diff
@@ -0,0 +1,329 @@
+# Transformer-Modelle - wozu sind sie imstande?
+
+<DocNotebookDropdown
```
Comment (lewtun) [inline chapters/de/chapter1/3.mdx:3]: I think this needs to be replaced with:

```
<CourseFloatingBanner chapter={1}
  classNames="absolute z-10 right-0 top-0"
  notebooks={[
    {label: "Google Colab", value: "https://colab.research.google.com/github/huggingface/notebooks/blob/master/course/en/chapter1/section3.ipynb"},
    {label: "Aws Studio", value: "https://studiolab.sagemaker.aws/import/github/huggingface/notebooks/blob/master/course/en/chapter1/section3.ipynb"},
]} />
```

**Review by lewtun — COMMENTED** [review]
Just left a few small comments about tweaking the meta information at the top of each section - otherwise, this looks great 🔥 !

**Comment by MarcusFra** [comment]
@lewtun Sure, thanks!

**Comment by MarcusFra** [comment]
@lewtun I added the banners and translated the additional faq section. Chapter 1 should be ready to be merged now.

**Review by lewtun — APPROVED** [review]
Thanks for iterating @MarcusFra - the changes look great 🔥 !

**Pull request merged**